lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -321,10 +321,10 @@ LogicalResult CreateSiFiveMetadataPass::emitSitestBlackboxMetadata() {
   // Any extmodule with these annotations or one of these ScalaClass classes
   // should be excluded from the blackbox list.
   std::array<StringRef, 3> classBlackList = {
-      "freechips.rocketchip.util.BlackBoxedROM", "chisel3.shim.CloneModule",
-      "sifive.enterprise.grandcentral.MemTap"};
```

Comment:
- `chisel3.shim.CloneModule` was removed from this pass upstream, it's uses were replaced with something else
- I'm not certain that `freechips.rocketchip.util.BlackBoxedROM` is actually an annotation, looks like it comes from here
https://github.com/chipsalliance/rocket-chip/blob/b503f8ac28a497b2463ffbac84bfe66533ace0bb/src/main/scala/util/ROMGenerator.scala#L10

Reply:
I think `memTapClass` is different from `*.MemTap` (the former is `*.MemTapAnnotation`).

Reply:
Woops yeah, I should have checked closer, none of these are actually annotations.